jobs/bosh-dns/spec

```diff
@@ -117,6 +117,14 @@ properties:
     description: "When enabled bosh-dns will cache recursor responses using the default coredns cache plugin"
     default: false
 
+  metrics.enabled:
```

Comment:
Can metrics be added to the windows job as well?

Reply:
The metrics should also work on Windows. I have added the configuration the the windows job. I don't have a Windows setup to test it.